src/app/view/tools/insertTextTool.js

```diff
@@ -40,31 +40,23 @@ export default class InsertTextTool extends MouseTool {
     }
 
     createNewFeature(point) {
-        // new Params(
-        //     {
-        //         position: PositionTool.getTarget(point),
-        //         height: 200
-        //     },
-        //     { position: "Point" },
-        //     { height: "Float", text: "String" }
-        // )
         let fixedpoint = PositionTool.getTarget(point);
         let newFeature = Device.makeFeature(
             "Text",
             {
                 position: fixedpoint,
                 height: 200,
                 text: this._text,
-                fontSize: this.fontSize
+                fontSize: this.fontSize * 10000
             },
             "TEXT_" + this._text,
             ComponentAPI.generateID(),
             "XY",
             null
         );
-        // this.currentFeatureID = newFeature.ID;
-        this.viewManagerDelegate.addFeature(newFeature);
-        //Registry.viewManager.renderLayers[Registry.viewManager.activeRenderLayer].addFeature(newFeature);
+        let physical = false;
+        Registry.viewManager.addFeature(newFeature, Registry.viewManager.activeRenderLayer, physical);
+        if (!physical) Registry.viewManager.view.addComponent("Text", newFeature.getParams(), [newFeature.ID], false);
```

Comment:
Need to take a look at this logic

Reply:
Modified this to not take on non-physical text , we will be using a different pipeline here